src/schemas/raster.schema.json

```diff
@@ -4,22 +4,71 @@
   "title": "Vitessce image data",
   "type": "object",
   "definitions": {
-    "raster": {
+    "imageTransform": {
       "type": "object",
-      "additionalProperties": true,
-      "required": ["sample", "tileSource"],
+      "additionalProperties": false,
+      "required": ["scale", "translate"],
       "properties": {
-        "sample": { "type": "number" },
-        "tileSource": {
-          "oneOf": [
-            { "type": "object" },
-            { "type": "string" }
-          ]
+        "scale": { "type": "integer" },
+        "translate": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["y", "x"],
+          "properties": {
+            "y": { "type": "integer" },
+            "x": { "type": "integer" }
+          }
         }
       }
+    },
+    "imageDimensions": {
+      "type": "array",
```

Comment:
Will the length typically be 2? Add a constraint for that? -- Or I may be misunderstanding the purpose.

Reply:
image dimensions are defined as and array of `{name, type, values}`. So for N-dimensional images `[time, z, channel, y, x]` the dimensions would be of length 5?